### PR TITLE
Finish localization support

### PR DESCRIPTION
**Pull request by labpaw**
Finished incomplete todos. Should be good but please double check it.

### CODE REVIEW

**Comment by labpaw** [comment]
Fixed conflict caused by https://github.com/DayZMod/DayZ/commit/1bdf8144d78bcd194a61ed82f49c21e115c7efa7 Ready to be merged.

File: SQF/dayz_code/stringtable.xml [hunk 1]
```diff
@@ -3731,6 +3739,34 @@
...
+		</Key>
+		<Key ID="STR_EQUIP_DESC_49">
+			<English>Azithromycin is a broad spectrum antibiotic used to treat or prevent certain bacterial infections. 5 Remaining.</English>
+			<Russian>Азитромицин это многоцелевой антибиотик, использующийся для лечения бактериальных инфекций. Осталось 5 таблеток.</Russian>
+		</Key>
+		<Key ID="STR_EQUIP_DESC_50">
+			<English>Azithromycin is a broad spectrum antibiotic used to treat or prevent certain bacterial infections. 4 Remaining.</English>
+			<Russian>Азитромицин это многоцелевой антибиотик, использующийся для лечения бактериальных инфекций. Осталось 4 таблетки.</Russian>
+		</Key>
+		<Key ID="STR_EQUIP_DESC_51">
```
Comment (ebayShopper) [inline SQF/dayz_code/stringtable.xml:3754]: should be 51, 52, 53, 54

File: SQF/dayz_code/actions/player_consume.sqf [hunk 2]
```diff
@@ -36,8 +36,7 @@ if ((getNumber (configFile >> "CfgMovesMaleSdr" >> "States" >> (animationState p
 //player doesn't have the consumable item
 if (!(_this in magazines player)) exitWith
 {
-	//TODO move to stringtable
-	"You seem to have misplaced the foodstuff." call dayz_rollingMessages;
+	(localize "$str_player_33") call dayz_rollingMessages;
```
Comment (ebayShopper) [inline SQF/dayz_code/actions/player_consume.sqf]: should be
`localize "str_player_33" call dayz_rollingMessages;`

File: SQF/dayz_code/stringtable.xml [hunk 3]
```diff
@@ -1005,6 +1005,14 @@
 			<Czech>Nalámat kámen</Czech>
 			<German>Steine abbauen</German>
 		</Key>
+		<Key ID="STR_ACTIONS_FIX_HATCHET">
+			<English>Починить топор</English>
```
Comment (ebayShopper) [inline SQF/dayz_code/stringtable.xml]: Russian

File: SQF/dayz_code/stringtable.xml [hunk 4]
```diff
@@ -2406,12 +2414,8 @@
 			<German>Ziegelstein</German>
 		</Key>
 		<Key ID="STR_ITEM_DESC_equip_brick">
-			<English>A brick used in crafting.</English>
```
Comment (ebayShopper) [inline SQF/dayz_code/stringtable.xml:2409]: not used

Comment (labpaw) [inline SQF/dayz_code/stringtable.xml:2409]: Yes, it's was replaced with new string.

File: SQF/dayz_code/actions/player_consume.sqf (hunk 2 above)
Comment (labpaw) [inline SQF/dayz_code/actions/player_consume.sqf]: Parentheses are used in DayZ code. Look for string above

**Comment by labpaw** [comment]
Updated

File: SQF/dayz_code/stringtable.xml [hunk 5]
```diff
@@ -5313,6 +5361,10 @@
 			<Russian>Армейский котелок для хранения жидкостей и еды.</Russian>
 			<German>Eine stabile Feldflasche, um Wasser zu transportieren.</German>
 		</Key>
+		<Key ID="STR_ITEM_CANTEEN_EMPTY">
+			<English>Empty</English>
```
Comment (ebayShopper) [inline SQF/dayz_code/stringtable.xml:5365]: duplicate of STR_EQUIP_NAME_13_EMPTY

Comment (labpaw) [inline SQF/dayz_code/stringtable.xml:5365]: Can we leave those strings as different? They a bit different in russian. I think it's not a big deal.

Comment (ebayShopper) [inline SQF/dayz_code/stringtable.xml:5365]: ok

**Pull request merged**